rust_dev_preview/iam/src/bin/iam-getting-started.rs

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+
+Purpose
+
+Shows how to use AWS SDK for PHP v3 to get started using AWS Identity and Access Management (IAM).
```

Comment:
use AWS SDK for PHP v3 --> use the AWS SDK for PHP (v3)

Reply:
fixed